packages/web3-wrapper/package.json

```diff
@@ -41,6 +46,13 @@
         "copyfiles": "^1.2.0",
         "npm-run-all": "^4.1.2",
         "shx": "^0.2.2",
+        "chai": "^4.0.1",
+        "mocha": "^4.0.1",
+        "nyc": "^11.0.1",
+        "ganache-core": "0xProject/ganache-core",
```

Comment:
Can we use the `ganacheSubprovider` and import `@0xproject/subproviders` rather then using `ganache-core` directly?

Reply:
We can't use the web3Factory because that will introducce a cycle in dependencies graph.
If we use Ganache subprovider then we need to create a provider engine, add a subprovider, start it. That doesn't make sense. Here I just get a provider and I'm good to go.